app/(Components)/(Form)/BotzEvent/VideoCategories.tsx

```diff
@@ -8,24 +8,33 @@ export const VideoCategories: React.FC<{
 	addCategory: (newCategory: string) => void;
 	removeCategory: (oldCategory: string) => void;
 }> = ({categoriesId, addCategory, removeCategory}) => {
+	const [maxSelected, setMaxSelected] = useState<boolean>(true);
 	const handleAddRemoveCategory = (checked: boolean, categoryId: string) => {
 		checked ? removeCategory(categoryId) : addCategory(categoryId);
 	};
 
+	useEffect(() => {
+		categoriesId.length < 4 ? setMaxSelected(false) : setMaxSelected(true);
+	}, [categoriesId]);
+
```

Comment:
You can remove the `useEffet` and the `useState` with another react hook 👀

Reply:
I used the hook `useMemo`, its better indeed 👍.